rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -2567,7 +2603,8 @@ private void validateNodeSourceNameOrFail(String nodeSourceName) {
         }
         if (this.definedNodeSources.containsKey(nodeSourceName) ||
             this.deployedNodeSources.containsKey(nodeSourceName) ||
-            this.delayedNodeSourceRemovalEvents.containsKey(nodeSourceName)) {
+            this.delayedNodeSourceRemovalEvents.containsKey(nodeSourceName) ||
```

Comment:
this check is a bit strange, is it meant to handle the following scenario ?

1) user removes a ns, waiting on busy nodes
2) It takes too long, user try again to remove the ns.

Reply:
@fviale did not get your point. This check here is to ensure we dont create new NS with a name that is already exist

Reply:
sorry, code folding misunderstanding